### PR TITLE
Fix Issue 15360 - typeof is not a template argument

### PR DESCRIPTION
**Pull request by RazvanN7**
Spec PR: https://github.com/dlang/dlang.org/pull/2387

### CODE REVIEW

File: src/dmd/parse.d [hunk 1]
```diff
@@ -2028,12 +2028,14 @@ final class Parser(AST) : Lexer
         case TOK.functionString:
         case TOK.prettyFunction:
         case TOK.this_:
-            {
-                // Template argument is an expression
-                AST.Expression ea = parsePrimaryExp();
-                tiargs.push(ea);
-                break;
-            }
```
Comment (RazvanN7) [inline src/dmd/parse.d]: This code block here was not respecting the norm that the other code blocks introduced, so I fixed the indentation.

**Comment by jacob-carlborg** [comment]
I think this bug should be closed as “invalid” or “won’t fix”.

**Comment by RazvanN7** [comment]
>I think this bug should be closed as “invalid” or “won’t fix”.

That was my first reaction also, but I thought it would be best to make a PR and discuss it. Closing PR and issue

**Comment by jacob-carlborg** [comment]
It can be discussed directly in the issue as well.

**Pull request closed without merging**